### DISCUSSION TITLE
Can someone update this plugin please?

### DISCUSSION THREAD

**Discussion by IchiKen98** [Pryaxis/TShock#1744] [General]
Hi there, i'm lokking for someone who can update this plugin, i will really appreciate.
Link: https://tshock.co/xf/index.php?resources/localized-chat.123/

**Comment by QuiCM** [comment]
Hi. I've done a quick reference update & version tick. [Let me know](https://github.com/QuiCM/Localized-Chat/issues/new/choose) if it doesn't work :)

(Download from [here](https://github.com/QuiCM/Localized-Chat/releases/tag/1.2))

**Reply by IchiKen98** [reply]
Many thanks dudeeeeeeeee <3
